provider/cmd/main.go

```diff
@@ -13,9 +12,10 @@ import (
 
 func main() {
 	ctx := ctrl.SetupSignalHandler()
+	mainLog := ctrl.Log.WithName("main")
```

Comment:
Just use `log` here as it is clear from the context what this logger is being used for.

Reply:
fixed